commons/src/main/java/io/aiven/kafka/connect/common/config/SourceConfigFragment.java

```diff
@@ -41,9 +46,14 @@ public static ConfigDef update(final ConfigDef configDef) {
         int sourcePollingConfigCounter = 0;
 
         configDef.define(MAX_POLL_RECORDS, ConfigDef.Type.INT, 500, ConfigDef.Range.atLeast(1),
-                ConfigDef.Importance.MEDIUM, "Max poll records", GROUP_OTHER, sourcePollingConfigCounter++, // NOPMD
-                // UnusedAssignment
+                ConfigDef.Importance.MEDIUM, "Max poll records", GROUP_OTHER, sourcePollingConfigCounter++,
                 ConfigDef.Width.NONE, MAX_POLL_RECORDS);
+        // KIP-298 Error Handling in Connect
+        configDef.define(ERRORS_TOLERANCE, ConfigDef.Type.STRING, "none", new ErrorsToleranceValidator(),
```

Comment:
with none as default, we ignore any sort of errors. correct ?

Reply:
none means no tolerance for errors and causes the Connector to crash, so users must decide if they want to ignore all errors by specifically setting this.

At the moment the framework only supports none and all